### PR TITLE
fix: finish sprint

### LINKED ISSUE #10: finish-sprint breaks because of false format error

### PR DESCRIPTION
**Pull request by paibamboo**
Closes #10

### CODE REVIEW

File: lib/git.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ async function finishSprint(config) {
       }
       const commitMessage = commit.substring(commit.indexOf(' ') + 1);
       if (commitMessage.indexOf('Merge pull request') !== -1) {
-        const issue = toIssueNumberAndName(commitMessage.substring(commitMessage.lastIndexOf(' ') + 1), owner + '/');
+        const issue = toIssueNumberAndName(commitMessage.substring(commitMessage.lastIndexOf('/') + 1));
```
Comment (adhocore) [inline lib/git.js]: `commitMessage.lastIndexOf('/') + 1` is still not failsafe
it really depends on the nature of free input - which is likely not in expected format?

Comment (paibamboo) [reply lib/git.js]: Format is `${projectName}-sprint-${sprintNumber}/${issueNumber}-${issueName}` and we don't expect `${issueName}` to contain another `/`

Comment (adhocore) [reply lib/git.js]: hmm but you are using commit msg?

Comment (paibamboo) [reply lib/git.js]: Merge commit message. Somethinkg like this
image

Comment (adhocore) [reply lib/git.js]: for this screenshot, what will be the output? `1848-add-linting` ?

Comment (paibamboo) [reply lib/git.js]:
```
{
  issueNumber: 1848,
  issueName: 'add linting'
}
```

**Pull request merged**